adadgio/inapp-sfr-ref-rhumato
# Feature/score/bvas

**description :**
add score for BVAS

:warning:  no css fo now

**How to test**

check if values are ok with model 
- array for stable symptomes (categorie.stable)
- array for new symptomes (categore.unstable)
- max for stable symptomes (categore.stableMax)
- max for new symptomes (categore.unstableMax)
- breaking points in total value for the gravity interpretation (11,30,50)

compute
- check sub totals / totals increase (to the categorie's max) when checked

catégorie maximum
- check the sub total value for a catégorie don't goes up to the max displayed
- check total is ok when max is reached in one or more categories

stable/new symptomes toggle
- check the max for each categorie is toogled
- check sub totals and total are updated
